app/models/legislator.rb

```diff
@@ -53,12 +53,13 @@ class Legislator < ActiveRecord::Base
-  MAP_LABELS = {"ME"=>[0.5, 0.5], "MN"=>[0.0, 0.0], "MI"=>[0.0, 0.5], "VT"=>[0.0, 0.5], "NH"=>[0.0, 0.0], "WI"=>[0.0, 0.5], "NY"=>[0.0, 0.5], "MA"=>[0.5, 0.5], "WA"=>[0.0, 0.0], "ID"=>[0.0, 0.0], "MT"=>[0.0, 0.5], "CT"=>[0.5, 0.5], "RI"=>[0.5, 0.5], "OR"=>[0.0, 0.0], "WY"=>[0.0, 0.5], "IN"=>[0.5, 0.5], "OH"=>[0.0, 0.5], "IL"=>[0.0, 0.5], "PA"=>[0.0, 0.5], "NJ"=>[0.5, 0.0], "WV"=>[0.5, 0.5], "MD"=>[0.5, 0.0], "CA"=>[0.5, 0.5], "NV"=>[0.5, 0.5], "ND"=>[0.0, 0.0], "IA"=>[0.5, 0.0], "DE"=>[0.0, 0.0], "SD"=>[0.0, 0.0], "KY"=>[0.0, 0.5], "VA"=>[0.5, 0.0], "NE"=>[0.5, 0.5], "UT"=>[0.5, 0.5], "KS"=>[0.5, 0.0], "MO"=>[0.5, 0.5], "CO"=>[0.0, 0.0], "TN"=>[0.5, 0.0], "NC"=>[0.5, 0.0], "OK"=>[0.5, 0.5], "NM"=>[0.5, 0.5], "SC"=>[0.5, 0.5], "AZ"=>[0.5, 0.0], "AR"=>[0.5, 0.0], "AL"=>[0.5, 0.5], "GA"=>[0.5, 0.5], "AK"=>[0.0, 0.5], "MS"=>[0.0, 0.5], "TX"=>[0.0, 0.0], "LA"=>[0.5, 0.0], "HI"=>[0.5, 0.0], "FL"=>[0.5, 0.5]}
+  MAP_LABELS = {"ME"=>[[34.5, 0.0], [34.5, 0.0]], "MN"=>[[13.0, 1.0], [13.0, 1.0]], "MI"=>[[19.0, 1.0], [19.5, 1.0]], "VT"=>[[31.0, 1.0], [31.5, 1.0]], "NH"=>[[33.0, 1.0], [33.0, 1.0]], "WI"=>[[16.0, 2.0], [16.5, 2.0]], "NY"=>[[28.0, 3.0], [28.5, 3.0]], "MA"=>[[33.5, 3.0], [33.5, 3.0]], "WA"=>[[2.0, 4.0], [2.0, 4.0]], "ID"=>[[5.0, 5.0], [5.0, 5.0]], "MT"=>[[7.0, 5.0], [7.5, 5.0]], "CT"=>[[31.5, 5.0], [31.5, 5.0]], "RI"=>[[34.5, 5.0], [34.5, 5.0]], "OR"=>[[2.0, 6.0], [2.0, 6.0]], "WY"=>[[8.0, 6.0], [8.5, 6.0]], "IN"=>[[19.5, 6.0], [19.5, 6.0]], "OH"=>[[22.0, 6.0], [22.5, 6.0]], "IL"=>[[17.0, 7.0], [17.5, 7.0]], "PA"=>[[26.0, 7.0], [26.5, 7.0]], "NJ"=>[[29.5, 9.0], [29.0, 9.0]], "WV"=>[[20.5, 10.0], [20.5, 10.0]], "MD"=>[[24.5, 10.0], [24.0, 10.0]], "CA"=>[[1.5, 11.0], [1.5, 11.0]], "NV"=>[[4.5, 11.0], [4.5, 11.0]], "ND"=>[[11.0, 11.0], [11.0, 11.0]], "IA"=>[[13.5, 11.0], [13.0, 11.0]], "DE"=>[[28.0, 11.0], [28.0, 11.0]], "SD"=>[[11.0, 12.0], [11.0, 12.0]], "KY"=>[[19.0, 12.0], [19.5, 12.0]], "VA"=>[[23.5, 12.0], [23.0, 12.0]], "NE"=>[[11.5, 13.0], [11.5, 13.0]], "UT"=>[[6.5, 14.0], [6.5, 14.0]], "KS"=>[[13.5, 14.0], [13.0, 14.0]], "MO"=>[[15.5, 14.0], [15.5, 14.0]], "CO"=>[[9.0, 15.0], [9.0, 15.0]], "TN"=>[[19.5, 15.0], [19.0, 15.0]], "NC"=>[[23.5, 15.0], [23.0, 15.0]], "OK"=>[[12.5, 16.0], [12.5, 16.0]], "NM"=>[[8.5, 17.0], [8.5, 17.0]], "SC"=>[[24.5, 17.0], [24.5, 17.0]], "AZ"=>[[6.5, 18.0], [6.0, 18.0]], "AR"=>[[15.5, 18.0], [15.0, 18.0]], "AL"=>[[18.5, 18.0], [18.5, 18.0]], "GA"=>[[21.5, 18.0], [21.5, 18.0]], "AK"=>[[1.0, 19.0], [1.5, 19.0]], "MS"=>[[17.0, 19.0], [17.5, 19.0]], "TX"=>[[11.0, 21.0], [11.0, 21.0]], "LA"=>[[15.5, 21.0], [15.0, 21.0]], "HI"=>[[4.5, 22.0], [4.0, 22.0]], "FL"=>[[23.5, 23.0], [23.5, 23.0]]}
```

Comment:
Line is too long. [1749/120]<br>Surrounding space missing for operator '=>'.<br>Space inside { missing.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.